Ref/SignalGen/test/ut/Tester.hpp

```diff
@@ -21,6 +21,8 @@ namespace Ref {
   class Tester :
     public SignalGenGTestBase
   {
+      // Maximum size of histories storing events, telemetry, and port outputs
+      static const NATIVE_INT_TYPE MAX_HISTORY_SIZE = 10;
```

Comment:
Since `TEST_INSTANCE_ID` and `TEST_INSTANCE_QUEUE_DEPTH` are not `#ifndef` in TesterHelper.cpp anymore (with your latest commit), we probably should define them here just like if it was generated - no? Otherwise it won't compile out of the box.

Reply:
yes.  Not sure why the compile worked for me.....

Reply:
Fixed!